plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -645,6 +646,16 @@ export class CatalogBuilder {
 
     await connectEntityProviders(providerDatabase, entityProviders);
```

Comment:
I wonder if it might be best to evict before we connect the new ones? That way any synchronous ingestion will avoid conflicting with orphaned entities, otherwise you'd need a second instance or restart to reconcile that

Reply:
Resolved in [abd0ab7](https://github.com/backstage/backstage/pull/28971/commits/abd0ab7901aff9539992c03e17764acd8622fc61)